lib/src/main/java/org/team100/lib/indicator/LEDIndicator.java

```diff
@@ -62,21 +63,41 @@ private State(Color color) {
     private final AddressableLEDBuffer buffer;
     private final AddressableLEDBuffer blackBuffer;
     private final List<LEDStrip> leds = new ArrayList<>();
-    private boolean flashing = false;
+    private boolean flashing;
+    private final int m_numstrips;
+    private final int m_numLEDs;
 
-    public LEDIndicator(int port, LEDStrip... strips) {
+    public LEDIndicator(int port, int numLEDs, LEDStrip... strips) {
+        int numstrips = 0;
+        m_numLEDs = numLEDs;
+        for (LEDStrip strip : strips) {
+            numstrips++;
+        }
+        m_numstrips = numstrips;
         Collections.addAll(leds, strips);
         int length = leds.stream().map(LEDStrip::getLength).reduce(0, Integer::sum);
         led = new AddressableLED(port);
         led.setLength(length);
         buffer = new AddressableLEDBuffer(length);
         blackBuffer = new AddressableLEDBuffer(length);
+        // for (int i = 0; i < m_numstrips; i++) {
+        //     Patterns.solid(leds.get(i * m_numLEDs + 1), blackBuffer, Color.kWhite);
+        // }
         led.setData(buffer);
         led.start();
     }
 
-    public void setStripSolid(int index, State s) {
-        Patterns.solid(leds.get(index), buffer, s.color);
+    public void setStripSolid(LEDStrip.strip state, State s) {
```

Comment:
the use of the word "state" in the first parameter is confusing to me.  what 'state' does this describe?